doc/source/Resources/Code_Guidelines.rst

```diff
@@ -215,11 +213,11 @@ a new section is starting.
 Maximum line length
 -------------------
 For source code lines, best practice is to keep the length at or below
-79 characters.  For docstrings and comments, best practice is to keep
+79 characters. For docstrings and comments, best practice is to keep
```

Comment:
120 is the line length specified in the pre-commit config.
```suggestion
120 characters. For docstrings and comments, best practice is to keep
```

Reply:
@MaxJPRey That entire Coding section is actually coming out of the PyAEDT doc in a PR later today. I'[ll make sure the PyAnsys Dev Guide has 120 characters--though I think it does.